sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/BlobCheckpointStore/BlobCheckpointStoreInternal.cs

```diff
@@ -501,24 +501,21 @@ private async Task UpdateCheckpointInternalAsync(string fullyQualifiedNamespace,
                                                           DateTimeOffset modifiedDate)
         {
             var startingPosition = default(EventPosition?);
-            var offset = default(long?);
+            string offset = null;
```

Comment:
```suggestion
            var offset = default(string);
```

nittiest of nits, for consistency with surrounding code.